tests/gpu-tests/test_generate.py

```diff
@@ -64,6 +64,46 @@ def test_vllm_generate_greedy():
         assert 'generation' in data
 
 
+@pytest.mark.gpu
+def test_vllm_reward():
```

Comment:
is this the same as what's in test_reward.py?

Reply:
Oops yeah, forgot to sync it back to the main branch's version!